spec/models/active_shipping_calculator_spec.rb

```diff
@@ -97,12 +87,8 @@
   describe 'compute' do
     subject { calculator.compute(package) }
 
-    let(:rates) do
-      [double('rate', service_name: 'Bogus Calculator', price: 999)]
-    end
-
     it 'should use the carrier supplied in the initializer' do
-      expect(carrier).to receive(:find_rates) { response }
+      expect(calculator.carrier).to be_an_instance_of(carrier.class)
```

Comment:
I think so too, I guess maybe you can test that the calculator is using the proper class instead of trying to initialize another carrier but I'm totally fine with removing it